src/main/java/appeng/me/storage/NetworkInventoryHandler.java

```diff
@@ -137,20 +135,20 @@ private boolean diveList( final NetworkInventoryHandler<T> networkInventoryHandl
 		return false;
 	}
 
-	private boolean testPermission( final BaseActionSource src, final SecurityPermissions permission )
+	private boolean testPermission( final IActionSource src, final SecurityPermissions permission )
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Refactor this method to not always return the same value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3516)